### PR TITLE
CDS-18 CANdevStudio shall contain toolbar

### PR DESCRIPTION
**Pull request by Mcko**

### CODE REVIEW

File: src/gui/CMakeLists.txt [hunk 1]
```diff
@@ -3,6 +3,7 @@
 set(srcs
     main.cpp
     mainwindow.cpp
+    toolbar/cdstoolbar.cpp
```
Comment (rkollataj) [inline src/gui/CMakeLists.txt]: Let's keep it in the same dir as main.cpp for now.

File: src/gui/mainwindow.h [hunk 2]
```diff
@@ -21,6 +24,9 @@ class MainWindow : public QMainWindow {
     ~MainWindow();
 
 private:
+    std::unique_ptr<QWidget> MainWidget;
```
Comment (rkollataj) [inline src/gui/mainwindow.h]: You probably took a example from below, but usage of unique_ptr is wrong here. Qt widget and windows took ownership of the pointer so we always should pass raw pointers for such cases. Anyway this is just a side note as these fields will disappear once you will rebase with master :)

File: src/gui/toolbar/cdstoolbar.cpp [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    cds_debug("start button clicked");
+    ui->startButton->setEnabled(false);
+    ui->stopButton->setEnabled(true);
+    emit simulationStart();
+}
+
+void CdsToolbar::stopButtonClicked()
+{
+    cds_debug("stop button clicked");
+    ui->stopButton->setEnabled(false);
```
Comment (rkollataj) [inline src/gui/toolbar/cdstoolbar.cpp]: Use this trick to handle enabling, disabling the buttons:
    connect(pbStart, &QPushButton::clicked, pbStop, &QPushButton::setDisabled);
    connect(pbStart, &QPushButton::clicked, pbStart, &QPushButton::setEnabled);

File: src/gui/toolbar/cdstoolbar.cpp [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    ui->startButton->setEnabled(true);
+    emit simulationStop();
```
Comment (rkollataj) [inline src/gui/toolbar/cdstoolbar.cpp]: If you connect "clicked" signal with "simulationStart" it will do the work. No need to emit explicitly.

**Comment by rkollataj** [comment]
Please add relevant JIRA ticket number to PR name.

File: src/gui/toolbar/cdstoolbar.cpp (hunk 4 above)
Comment (Mcko) [inline src/gui/toolbar/cdstoolbar.cpp]: Applied changes suggested in comments

**Pull request closed without merging**